.github/workflows/python-tests/action.yml

```diff
@@ -31,10 +31,21 @@ runs:
       run: tox -e docs-check
       shell: bash
       working-directory: ${{ inputs.path }}
-    - name: Run tox python 3.8 unit tests
+    - name: Run tox python3.8 unit tests
+      if: ${{ inputs.python-version == '3.8'}}
       run: tox -e py38
       shell: bash
       working-directory: ${{ inputs.path }}
+    - name: Run tox python3.9 unit tests
```

Comment:
It seems like there'd be a way to parameterize the python verson from the matrix so we don't have to repeat this

Reply:
eg, just one step like: 
`  - name: Run tox python ${{matrix.version}} unit tests
      run: tox -e ${{matrix.version}}
      shell: bash
      working-directory: ${{ inputs.path }}`

Reply:
Good point.  I think you would have to convert the python version from 3.8 to py38 for tox though.  So there is some type of expression you would need to write to convert to the tox format.

Reply:
I think I can do something close to it with includes in GitHub actions. 
https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

Reply:
I think this is still needed.  It’d be nice to have one action rather than 3 for each version.